app/javascript/dashboard/store/modules/contacts/actions.js

```diff
@@ -83,6 +83,21 @@ export const actions = {
     }
   },
 
+  import: async ({ commit }, file) => {
```

Comment:
Please write test cases for this.

Reply:
Test case for the action in https://github.com/chatwoot/chatwoot/pull/3026/commits/fa5cc63f72a7bc8a86a1c010b95d09987e781014